vkcs/backup/data_source_vkcs_backup_plan.go

```diff
@@ -0,0 +1,257 @@
+	GFSRetention      *PlanResourceGFSRetentionModel  `tfsdk:"gfs_retention"`
+	IncrementalBackup types.Bool                      `tfsdk:"incremental_backup"`
+	ProviderID        types.String                    `tfsdk:"provider_id"`
+	InstanceIDs       []types.String                  `tfsdk:"instance_ids"`
+	Region            types.String                    `tfsdk:"region"`
+}
+
+func (d *PlanDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed:    true,
+				Description: "ID of the resource",
+			},
+
+			"name": schema.StringAttribute{
+				Optional:    true,
+				Description: "Name of the backup plan",
+				Validators: []validator.String{
+					stringvalidator.AtLeastOneOf(path.Expressions{
+						path.MatchRoot("instance_id"),
+					}...),
+				},
+			},
+
+			"instance_id": schema.StringAttribute{
+				Optional:    true,
+				Description: "ID of the instance that should be included in backup plan",
+				Validators: []validator.String{
+					stringvalidator.AtLeastOneOf(path.Expressions{
+						path.MatchRoot("name"),
+					}...),
+				},
+			},
+
+			"schedule": schema.SingleNestedAttribute{
+				Computed: true,
+				Attributes: map[string]schema.Attribute{
+					"date": schema.ListAttribute{
+						ElementType: types.StringType,
+						Computed:    true,
+						Description: "List of days when to perform backups. If incremental_backups is enabled, this field contains day of full backup",
+					},
+					"time": schema.StringAttribute{
+						Computed:    true,
+						Description: "Time of backup in format hh:mm, using UTC timezone",
+					},
+					"every_hours": schema.Int64Attribute{
+						Computed:    true,
+						Description: "Hour period of backups",
+					},
+				},
+			},
+
+			"full_retention": schema.SingleNestedAttribute{
+				Computed: true,
+				Attributes: map[string]schema.Attribute{
+					"max_full_backup": schema.Int64Attribute{
+						Computed:    true,
+						Description: "Maximum number of backups",
+					},
+				},
+				Description: "Parameters for full retention policy",
+			},
+
+			"gfs_retention": schema.SingleNestedAttribute{
+				Computed: true,
+				Attributes: map[string]schema.Attribute{
+					"gfs_weekly": schema.Int64Attribute{
+						Computed:    true,
+						Description: "Number of weeks to store backups",
+					},
+					"gfs_monthly": schema.Int64Attribute{
+						Computed:    true,
+						Description: "Number of months to store backups",
+					},
+					"gfs_yearly": schema.Int64Attribute{
+						Computed:    true,
+						Description: "Number of years to store backups",
+					},
+				},
+				Description: "Parameters for gfs retention policy",
+			},
+
+			"incremental_backup": schema.BoolAttribute{
+				Computed:    true,
+				Description: "Whether incremental backups should be stored",
```

Comment:
impossible to understand incremental backup politics

Reply:
Expanded descriptions for incremental backups and retention policies